content/applications/hr/appraisals/reporting.rst

```diff
@@ -0,0 +1,188 @@
+- :guilabel:`Current Skill Progress`: the current percentage of competency achieved for the skill.
+- :guilabel:`Justification`: any notes entered on the skill, explaining the progress.
+
+The color of the skill text indicates any changes from the previous appraisal. Skill levels that
+have increased since the last appraisal appear in green, skill levels that have not changed appear
+in black, and skills that have *regressed* appear in red.
+
+.. image:: reporting/skills-report.png
+   :align: center
+   :alt: A report showing all the skills grouped by employee.
+
+Filter skills
+~~~~~~~~~~~~~
+
+It is possible to only view skills that have changed over time, by using one of the preconfigured
+filters for the :guilabel:`Appraisal Skills Report`.
+
+To view the available filters, click the  :icon:`fa-caret-down` :guilabel:`(caret down)` icon in the
+:guilabel:`Search...` bar.
+
+.. image:: reporting/filter.png
+   :align: center
+   :alt: A report showing all the skills grouped by employee.
+
+In the :guilabel:`Filters` section on the left-side of the resulting drop-down menu, three options
+are presented:
+
+- :guilabel:`Improvement`: select this filter to **only** present skills that have increased since
+  the last appraisal. All skills appearing with this filter have green text, indicating the skill
+  level has improved.
+- :guilabel:`No Change`: select this filter to **only** present skills that have remained at the
+  same skill level since the last appraisal. All skills appearing with this filter have black text,
+  indicating there has been no change in the skill level.
+- :guilabel:`Regression`: select this filter to **only** present skills that have decreased since
+  the last appraisal. All skills appearing with this filter have red text, indicating the skill
+  level has dropped.
+
+It is possible to select as many or as few filters as desired.
+
+Group skills
+~~~~~~~~~~~~
+
+The :guilabel:`Appraisal Skills Report` is grouped by two categories, :guilabel:`Employee` and
+:guilabel:`Skill Type`, by default. These groupings can be removed, and/or different groupings can
+be added.
+
+To view the available grouping options, click the :icon:`fa-caret-down` :guilabel:`(caret down)`
+icon in the :guilabel:`Search...` bar.
+
+Active groupings have a green :icon:`fa-check` :guilabel:`(checkmark)` next to them. Click on a
+grouping option to change its current status, either from active to inactive (or vice versa). Any
+configuration of groupings can be selected.
+
+The available default :guilabel:`Group By` options are:
+
+- :guilabel:`Department`: groups all skills by department.
+- :guilabel:`Employee`: groups all skills by employee.
+- :guilabel:`Skill Type`: groups all skills by individual skill types.
+- :guilabel:`Evolution`: groups all skills by the following evolutions: :guilabel:`Decline`,
+  :guilabel:`Improvement`, :guilabel:`Just Added`, and :guilabel:`Same`.
```

Comment:
Same reasoning as above, this can all just be linked to the essentials content
```suggestion
The report can have other :ref:`filters <search/filters>` and :ref:`groupings <search/group>` set in
the :guilabel:`Search...` bar at the top.

.. seealso::
   - :doc:`Odoo essentials reporting <../../essentials/reporting>`
   - :doc:`../../essentials/search`
```